### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by shurichmond**
In this Project, I am adding a contact Form  to my portfolio;
- [x] Created the contact Form
- [x] Form Collects data by using the [Formspree service](https://formspree.io/html).
- [x] Used Flexbox lay out the elements on the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -226,6 +229,18 @@ <h1 class="title">About Myself</h1>
             </div>
         </div>
     </section>
+
+    <section class="contact-form">
+        <h1 class="form-header">Contact me</h1>
```
Comment (mckent05) [inline index.html:234]: **OPTIONAL** - To help improve the SEO of your site, I advise that you have only one `h1` tag on your HTML page. Please fix this 🙏

**Review by mckent05 — APPROVED** [review]
Hi @shurichmond, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job building your contact form 👍 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (shurichmond) [reply index.html:234]: So Only one h1 tag on My HTML page, meaning all the other h1 in the about section and form sections i used should be replaced?

Comment (nicupop729) [reply index.html:234]: Yes, it means exactly this :+1:

**Pull request merged**